assembly/compliance-tool/src/site/markdown/updater.md

```diff
@@ -0,0 +1,28 @@
+
+## <a name="SW360Updater">SW360 Updater</a>
+
+The SW360Updater takes a given csv file with release
+information and maps the contents on SW360Release objects
+that are posted into the given SW360 instance. 
+It has the ability to update release information of already existing releases. 
+
+### Properties
+- `targetDir`: Target directory in which artificial clearing reports are created in. 
+- `sourcesDirectory`: Directory where the sources for uploading are stored
+- `basedir`: Base directory of the execution
+- `csvFilePath`: Path and name to the csv file with the release information
+- `delimiter`: Delimiter used in the csv file to separate columns (by default it is `,`)
+- `encoding`: Encoding of the csv file, normally `UTF-8` 
+- `removeClearedSources`: A boolean property that controls whether the updater should remove the source attachments of a release from the local sources directory once the release has been cleared. Cleared releases are no longer relevant for the workflow of the compliance tool; so by setting this property to *true*, an automatic cleanup of the sources directory can be enabled. The default value is *false*.
+- `removeClearingDocuments`: A boolean property that controls whether clearing documents for releases should be removed after the release has been cleared. The default value is *false*.
+- `proxyHost`: If a proxy is in use, supply the host name
```

Comment:
Idea: To get rid off duplication, should the properties common to all execution modes (e.g. SW360 connection details) be described on the top-level site for the compliance tool?